packages/web-components/@sfx/search-box/stories/index.ts

```diff
@@ -1,68 +1,78 @@
-  console.log('event', e);
-});
+const searchboxNotesMarkdownIntro = ` # SF-X Search Box Component
 
-window.addEventListener(SEARCHBOX_EVENT.SEARCHBOX_CLICK, e => {
-  console.log('search box clicked event received');
-  console.log('event', e);
-});
+[SF-X Search Box README](https://github.com/groupby/sfx-view/tree/master/packages/web-components/%40sfx/search-box "SF-X Search Box README").
 
-window.addEventListener(SEARCHBOX_EVENT.SEARCHBOX_CHANGE, e => {
-  console.log('search box change event received');
-  console.log('event', e);
-});
-// end of event dispatch testing
+\`\`\`html
+<sfx-search-box
+placeholder="Search Here"
+searchbutton
+clearbutton
+></sfx-search-box>
+\`\`\`
+
+## Demonstrated in this story:`;
+
+function getSearchBoxComponent(): string {
+  const placeholder = text('Placeholder Text', 'Type your search');
+  const showSearchButton = boolean('Show search button', true) ? 'searchbutton' : '';
+  const showClearButton = boolean('Show clear button', true) ? 'clearbutton' : '';
+  return (
+    '<sfx-search-box\n' +
+    (placeholder ? ` placeholder="${placeholder}"\n` : '') +
+    (showSearchButton ? ` ${showSearchButton}\n` : '') +
+    (showClearButton ? ` ${showClearButton}\n` : '') +
+    '></sfx-search-box>'
+  );
+}
 
 storiesOf('Components|Searchbox', module)
   .addDecorator(withKnobs)
-    .add(
-      'Default - with search and clear button',
-      () => `
-    <sfx-search-box searchbutton clearbutton></sfx-search-box>
-    <button @click="${this.updateTextEvent}">
-      Click to dispatch update search term event
-    </button>
-  `
-    )
-    .add(
-      'Without any SFX provided buttons',
-      () => `
-    <sfx-search-box></sfx-search-box>
-    <button @click="${this.updateTextEvent}">
-      Click to dispatch update search term event
-    </button>
-  `
-    )
-    .add(
-      'With a clear button, without a search button',
-      () => `
-    <sfx-search-box clearbutton></sfx-search-box>
-    <button @click="${this.updateTextEvent}">
-      Click to dispatch update search term event
-    </button>
-  `
-    )
-    .add(
-      'With custom placeholder text',
-      () => `
-    <sfx-search-box placeholder="${text(
-      'Placeholder Title',
-      'Placeholder here...'
-    )}"></sfx-search-box>
-    <button @click="${this.updateTextEvent}">
-      Click to dispatch update search term event
-    </button>
-  `
-    );
+  .add(
+    'Rendering based on events and attributes',
+    () => {
+      const searchBoxComponent = getSearchBoxComponent();
+      return `
+      ${searchBoxComponent}
+      ${getDisplayCode(searchBoxComponent)}
+      `;
+    },
+    {
+      customEvents: updateTextEvent,
+      notes: {
+        markdown: `
+        ${searchboxNotesMarkdownIntro}
+
+          * The SF-X Search Box component rendering with customizations.
+            * Clear and search buttons are optional.
+              * If the 'clearbutton' attribute is present, a clear button will display.
```

Comment:
As a reminder, attribute names should be in backticks. This continues to apply to the rest of the PR.

```suggestion
              * If the \`clearbutton\` attribute is present, a clear button will display.
```